kie-wb/kie-wb-webapp/src/test/java/org/kie/workbench/security/WorkbenchACLTest.java

```diff
@@ -100,9 +100,7 @@
     static final List<String> USER_GRANTED = Arrays.asList(
             "perspective.read.SocialHomePagePerspective",
             "perspective.read.UserHomePagePerspective",
-            "perspective.read.Process Definition",
```

Comment:
@nmirasch not sure I understood why you had to remove the process definition and process instance perspectives. I would expect that to be granted to the user role as they are defined in the `security-module-user.properties ` file that you just fixed.